src/saml2/client_base.py

```diff
@@ -194,15 +193,14 @@ def __init__(self, config=None, identity_cache=None, state_cache=None,
                 self.want_assertions_or_response_signed,
             ]
         ):
-            warn_msg = (
+            error_msg = (
                 "The SAML service provider accepts "
                 "unsigned SAML Responses and Assertions. "
                 "This configuration is insecure. "
```

Comment:
Reordering the phrases to put emphasis on _insecure_.

```suggestion
                "This configuration is insecure. "
                "The SAML service provider accepts "
                "unsigned SAML Responses and Assertions. "
```